cmake-init/templates/common/CMakePresets.json

```diff
@@ -45,10 +45,18 @@
       }
     },{% end %}{% if conan %}
     {
-      "name": "conan",
+      "name": "conan-dev",
       "hidden": true,
       "cacheVariables": {
-        "CMAKE_TOOLCHAIN_FILE": "${sourceDir}/conan/conan_toolchain.cmake",
+        "CMAKE_TOOLCHAIN_FILE": "${sourceDir}/conan/debug/conan_toolchain.cmake",
+        "CMAKE_POLICY_DEFAULT_CMP0091": "NEW"
+      }
+    },
+    {
+      "name": "conan-rel",
+      "hidden": true,
+      "cacheVariables": {
+        "CMAKE_TOOLCHAIN_FILE": "${sourceDir}/conan/release/conan_toolchain.cmake",
```

Comment:
CMakePresets.json is more geared towards common parameters between local and CI environments. This is used nowhere in the CI, so it doesn't belong in here.

Same goes for the rest of the file, plus the inconsistent formatting is no good.